### ISSUE TITLE
Set Channel Name and Value at runtime

### ISSUE THREAD

**Issue by HuBDrill** [NIVeriStandAdd-Ons/VeriStand-steps-for-TestStand#50]
Hello Everyone, 

can someone please show me how to set the channel Names during runtime. 
I am doing an automated Test and load all the signals, Imports and parameters. During my Test I want to search the string array for e.g. Engine On and set its value to e.g. 1. Because this is an automated Tests, different parameters and signals would be set at different times. 
With my set up so far, I can initialize veristand and set the channel names manually and run and it works however to configure the Set Channel to get the names automatically has be a challenge.

Any replies with screenshot or examples would be great.
I appreciate your assistance.

**Comment by adchurch** [comment]
@HuBDrill if this is for a new project I would be sure to take a look at NI's new ASAM XIL Step Types: http://www.ni.com/product-documentation/55126/en/

It has been a while, but, regarding your question, note that the Get Channel Value and Set Channel Value step types can be configured two ways - either manually using the System Explorer browser tree or by setting the Step's properties directly. You can set step properties programmatically using a TestStand expression. For example you can set the channel name property of a Get Channel Value Step using a preceding expression step.

**Comment by HuBDrill** [comment]
@adchurch , yes it is a new project. Thanks for the ASAM XIL info. I will download it and take a look at it. 
Using the TS property expression to set the a Channel Value works in my setup. 
My Question is : e.g. 
1. I have a Model with Parameters (Testtime, EngineState,RPMState) and Signals (EngineOn, EngineOff, RPM). 
2. My TS expression has e.g. Local.searchString = EngineOn.
3. Using the AddOn Steps, during runtime I want to look up my Signals in search for "EngineOn" ? 
4. Using the AddOn Steps, during runtime I want to do get Parameter "TestTime" from the list and set it to value. 
5. If this Signal element is found, I can then set it to my wish value.

Basically, I want to get my parameter or signal names during runtime and then I can set their values.

Thanks
